### ISSUE TITLE
ERROR Failed to create uinput device: CError: code: 1 message: error: failed to open /dev/uinput

### ISSUE THREAD

**Issue by 2junho** [H-M-H/Weylus#41]
Hello,

I am trying to use Weylus in my Ubuntu 20.04 + iPad.

But I am facing an error message saying "Failed to create uinput device!" as I scan QR-code in the laptop using my iPad camera. I have googled to find a solution, but I can't find it yet.

Could you provide some help to resolve this?

Best wishes,
Jun-Ho

**Comment by H-M-H** [comment]
Hello,

have you made sure to follow the instructions here: https://github.com/H-M-H/Weylus#linux ? By default access to uinput is disabled for non root users.

**Comment by H-M-H** [comment]
It is possible to annotate PDFs with Xournal++.

**Comment by 2junho** [comment]
Yeah, I found it, it is exactly what I am looking for. Thanks a lot!
